ERROR:root:Unexcepted Error
Traceback (most recent call last):
  File "d:\APM_SEMI_MAP\tool.py", line 135, in accept
    row_data_lines, wafer_number, rowct, colct, lot = read_bak_file(filepath)
                                                      ^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\APM_SEMI_MAP\tool.py", line 154, in read_bak_file
    wafer_number = [line.split(':')[1].strip() for line in lines if line.startswith('WAFER')][0]
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
ERROR:root:Unexcepted Error
Traceback (most recent call last):
  File "d:\APM_SEMI_MAP\tool.py", line 135, in accept
    row_data_lines, wafer_number, rowct, colct, lot = read_bak_file(filepath)
                                                      ^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\APM_SEMI_MAP\tool.py", line 154, in read_bak_file
    wafer_number = [line.split(':')[1].strip() for line in lines if line.startswith('WAFER')][0]
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
ERROR:root:Unexcepted Error
Traceback (most recent call last):
  File "d:\APM_SEMI_MAP\tool.py", line 135, in accept
    row_data_lines, wafer_number, rowct, colct, lot = read_bak_file(filepath)
                                                      ^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\APM_SEMI_MAP\tool.py", line 154, in read_bak_file
    wafer_number = [line.split(':')[1].strip() for line in lines if line.startswith('WAFER')][0]
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
